react/tic-tac-toe/src/utils/Validations/index.js

```diff
@@ -0,0 +1,10 @@
+// eslint-disable-next-line no-useless-escape
+const VALID_EMAIL = /^\w+([\.-]?\w+)*@wolox.com/;
+
+export const isRequired = value => value ? undefined : 'Required field';
+
+export const hasLength = value =>
+  value.length < 8 ? 'Value length has to be greater than 8 characters' : undefined;
+
+export const isValidEmail = value =>
```

Comment:
I think this wasn't working correctly, don't know if you fixed it

Reply:
yes it's done, it was using `' '` instead of `undefined` what was causing trouble